pallets/handles/src/handles-utils/src/validator.rs

```diff
@@ -10,7 +10,21 @@ use constants::ALLOWED_UNICODE_CHARACTER_RANGES;
 
 /// Reserved words that cannot be used as the handle.
 const RESERVED_WORDS: [&str; 8] =
-	["admin", "everyone", "all", "administrator", "mod", "moderator", "here", "channel"];
+	["adm1n", "every0ne", "a11", "adm1n1strat0r", "m0d", "m0derat0r", "here", "channe1"];
```

Comment:
Oddly enough, the best way to do this was to do the conversion statically.

Note that `convert_to_canonical` cannot be used here as it cannot be converted into a const fn

Reply:
This is a bit confusing since we are using these keywords with like the following pub fn 
```rust
is_reserved_canonical_handle(input_str: &str) -> bool {
	RESERVED_WORDS.contains(&input_str)
}
```
this would only work if we convert all 
o -> 0
l -> 1 
and ...
which doesn't look like the canonical form.

Reply:
Well it looks like that's what we do which for a canonical form seems confusing.

Reply:
In retrospect canonical might have been better named something else, but yes, canonical is the trimmed, stripped, converted, etc... version